prov/util/src/util_profile.c

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (c) 2023, Intel Corporation. All rights reserved.
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the
+ * BSD license below:
+ *
+ *     Redistribution and use in source and binary forms, with or
+ *     without modification, are permitted provided that the following
+ *     conditions are met:
+ *
+ *      - Redistributions of source code must retain the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer.
+ *
+ *      - Redistributions in binary form must reproduce the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <ofi.h>
+#include <ofi_profile.h>
+
+#define PROF_LIST_SIZE	64
+
+#define OFI_PROF_DESC_SET(dest, src)                 \
+	do {                                             \
+		(dest)->id = (src)->id;                      \
+		(dest)->datatype_sel = (src)->datatype_sel;  \
+		(dest)->datatype = (src)->datatype;          \
+		(dest)->flags = (src)->flags;                \
+		(dest)->size = (src)->size;                  \
+		(dest)->name = (src)->name;                  \
+		(dest)->desc = (src)->desc;                  \
+	} while (0)
+
+struct fi_profile_desc  ofi_common_vars[] = {
+	{.id = FI_VAR_UNEXP_MSG_CNT,
```

Comment:
Add a new line after `{`, and indent the enclosed lines by a tab instead of a space. Same for other similar places.